### PR TITLE
Creating a file for Shahrekord, Chaharmahal-O-Bakhtiari, Iran

### LINKED ISSUE #50: Create Hamid914 hometown file
## Shahrekord, Chaharmahal-O-Bakhtiari, Iran
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [x] Have a conversation
- [x] Get a :+1:
- [x] :ship: it (a.k.a merge)

### PR DESCRIPTION
**Pull request by hamid914**
This PR resolves #50

### CODE REVIEW

File: hamid914-shahrekord.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Shahrekord, Chaharmahal-O-Bakhtiari
+
```
Comment (gerryderry) [inline hamid914-shahrekord.md:3]: sounds like a great place

File: hamid914-shahrekord.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Shahrekord, Chaharmahal-O-Bakhtiari
```
Comment (crazy4wisdom) [inline hamid914-shahrekord.md:1]: What country is Shahrekord located in?

File: hamid914-shahrekord.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+### Places to eat
```
Comment (githubteacher) [inline hamid914-shahrekord.md:4]: @hamid914 super excited to see you in class today! What you recommend as restaurants and foods someone "must try" while they are in town?

File: hamid914-shahrekord.md (hunk 1 above)
Comment (hamid914) [inline hamid914-shahrekord.md:3]: @gerryderry Thanks 😉

File: hamid914-shahrekord.md (hunk 2 above)
Comment (hamid914) [inline hamid914-shahrekord.md:1]: Shahrekord city is located in Chahrmahal-O-Bakhtiari state that is located in Iran.
Look at WikiPedia [Shahr-E Kord](https://en.wikipedia.org/wiki/Shahr-e_Kord)

File: hamid914-shahrekord.md (hunk 3 above)
Comment (hamid914) [inline hamid914-shahrekord.md:4]: @crichID Thanks 😄 I'm super excited being in the class! I recommend [Dizi](https://en.wikipedia.org/wiki/Abgoosht) in http://www.mrdizi.com) 👍 It is "must try" one

**Pull request merged**